spec/presenters/idv/in_person/ready_to_verify_presenter_spec.rb

```diff
@@ -159,18 +159,42 @@
     end
   end
 
-  describe '#usps_outage_message_enabled' do
-    subject(:usps_outage_message_enabled) { presenter.usps_outage_message_enabled }
+  describe '#date' do
+    let(:in_person_outage_expected_update_date) { 'January 1, 2024' }
+    subject(:date) { presenter.date }
+
+    it 'returns a formatted date for expected update after an outage' do
+      allow(IdentityConfig.store).to receive(:in_person_outage_expected_update_date).
+        and_return(in_person_outage_expected_update_date)
+      expect(date).to eq 'Monday, January 1'
+    end
+  end
+
+  describe '#email_date' do
+    let(:in_person_outage_emailed_by_date) { 'January 2, 2024' }
+    subject(:email_date) { presenter.email_date }
+
+    it 'returns a formatted email date' do
+      allow(IdentityConfig.store).to receive(:in_person_outage_emailed_by_date).
+        and_return(in_person_outage_emailed_by_date)
+      expect(email_date).to eq 'Tuesday, January 2'
```

Comment:
Should these be using translation files (for Tuesday and January) rather than hard coded? Same for above on line 169

Reply:
Ignore my suggestion. Thanks the reason